src/scope-for-gradle.ts

```diff
@@ -2,18 +2,15 @@ import * as core from '@actions/core'
 import * as executor from './executor'
 import * as exec from "@actions/exec";
 
-const SCOPE_AGENT_VERSION = "0.3.0"
-const SCOPE_GRADLE_PLUGIN_VERSION = "0.1.1-beta.1"
-
 async function run() {
     try {
-        let dsn = core.getInput("dsn", {required: true})
+        let dsn = core.getInput("dsn", {required: true});
         core.exportVariable("SCOPE_DSN", dsn);
 
         let executeTestPhase = core.getInput("run-tests", {required: true});
         let command = core.getInput("command", {required: true});
 
-        await executor.instrument(SCOPE_AGENT_VERSION, SCOPE_GRADLE_PLUGIN_VERSION);
+        await executor.instrument(false);
```

Comment:
will `allowBeta` ever be `true` ? Or is that some feature we want to add later on?

Reply:
Probably I'll have a branch for beta agents that it won't be public.